Stresser/RuleController.cpp

```diff
@@ -1,5 +1,7 @@
 #include "RuleController.h"
 
+#define RULE_ENTITY_PARAMS std::string id, std::string name, std::string type, std::string data
```

Comment:
This is what we've talked about yesterday.
I don't want from the controller to "know" RuleEntity inner build.
I'm ONLY allowed it to "know" how to convert JSON to RuleEntity and vice versa.
If we decide to CHANGE the inner build of RuleEntity we will also have to change RuleController - violates Open/Close principle.